### PR TITLE
Translating the Project to TypeScript

### PR DESCRIPTION
**Pull request by DrozdovDenys**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,11 +2,11 @@
   "name": "node_js_app",
   "version": "1.0.0",
   "description": "node.js app",
-  "main": "src/index.js",
+  "main": "src/index.ts",
   "type": "module",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "nodemon --watch src"
+    "start:dev": "npx nodemon"
```
Comment (KryvDanylo) [inline package.json]: So, if you have start:dev you should have build and start scripts for prod.

File: package.json [hunk 2]
```diff
@@ -2,11 +2,11 @@
   "name": "node_js_app",
   "version": "1.0.0",
   "description": "node.js app",
-  "main": "src/index.js",
+  "main": "src/index.ts",
```
Comment (KryvDanylo) [inline package.json]: The main file always should be a JS file. In your case, it's build/index.js it depends on what is your outDir in tsconfig.

**Review by KryvDanylo — CHANGES_REQUESTED** [review]

**Review by KryvDanylo — CHANGES_REQUESTED** [review]
Also, I didn't see linters it was 4 point of your HW requirements.

File: package.json (hunk 2 above)
Comment (DrozdovDenys) [reply package.json]: fixed

File: package.json (hunk 1 above)
Comment (DrozdovDenys) [reply package.json]: fixed

File: .eslintrc.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
+    "env": {
+        "browser": true,
```
Comment (KryvDanylo) [inline .eslintrc.json]: You work with node.js, not a browser.

**Review by KryvDanylo — CHANGES_REQUESTED** [review]

File: .eslintrc.json (hunk 3 above)
Comment (DrozdovDenys) [reply .eslintrc.json]: fixed

**Review by KryvDanylo — APPROVED** [review]

**Pull request merged**